### DISCUSSION TITLE
Basics of espressopp

### DISCUSSION THREAD

**Discussion by cjoydip** [espressopp/espressopp#416] [Q&A] [answered]
I need to implement espressopp for my new project. But, I am very new to the linux environments, python and espressopp software package in general. I know that espressopp is installed in my system. But I don't know how to check which version is installed via the terminal. Also, I have downloaded the documentation but am unable to run any of the sample scripts in the tutorials. Could anyone suggest me how to proceed in learning espressopp? Just to be clear the documentation does not help a beginner like me at all. Because if I run the python script of lennard jones potential tutorial the terminal shows error in the "import espressopp" command itself. I know in which directory the software is installed. From there onwards How do I proceed? Please help me in a plain and understandable manner.

**Comment by junghans** [comment, marked as answer]
@stuehn that is a question for you!

**Reply by stuehn** [reply]
If espressopp cannot be imported the PYHTONPATH environment variable may not be set correctly. 
When using a self compiled espressopp version make sure to source the ESPRC script residing in the espressopp installation folder.
When using a preinstalled or systemwide version please check with your system adminstrator to get the correct path to the python modules.
To get all details about the version of the imported espressopp module please use:
~~~
espressopp.Version().info()
~~~
